praveglet/src/main.rs

```diff
@@ -0,0 +1,199 @@
+/*
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+use pravega_client::client_factory::ClientFactory;
+use pravega_client_config::ClientConfigBuilder;
+use pravega_client_shared::{PravegaNodeUri, ScaleType, Scaling, Scope, ScopedStream, StreamConfiguration};
+use std::io::{self, BufRead};
+use structopt::StructOpt;
+use tracing::info;
+
+#[derive(StructOpt, Debug)]
+enum Command {
+    /// Create Scope    
+    CreateScope {
+        #[structopt(help = "controller-cli create-scope scope-name")]
+        scope_name: String,
+    },
+    /// Create Stream with a fixed segment count.
+    CreateStream {
+        #[structopt(help = "Scope Name")]
+        scope_name: String,
+        #[structopt(help = "Stream Name")]
+        stream_name: String,
+        #[structopt(help = "Segment Count")]
+        segment_count: i32,
+        #[structopt(help = "tag", value_name = "Tag,", use_delimiter = true, min_values = 0)]
+        tags: Vec<String>,
+    },
+    /// Write events from STDIN.
```

Comment:
This needs more documentation. 
Specifically where event boundaries occur.

Reply:
improved it.